fedimint-testing/Cargo.toml

```diff
@@ -30,6 +30,7 @@ ln-gateway = { path = "../gateway/ln-gateway" }
 futures = "0.3"
 lightning = "0.0.113"
 lightning-invoice = "0.21.0"
+project-root = "0.2.2"
```

Comment:
no, use `fedimint-testing`

Reply:
I think better solution is introduce a new variable $FM_SRC_DIR, this will be useful when testing external fedimintds #1987

Reply:
ok, I'll do that

Reply:
@Maan2003  Can I use FM_SRC_DIR=`dirname $PWD` in `scripts/build.sh` ?